src/WrapperApp/components/Simulation/RunSimulationForm.tsx

```diff
@@ -232,12 +232,24 @@ export function RunSimulationForm({
 						value={nTasks}
 						onChange={e => setNTasks(Math.max(1, parseInt(e.target.value)))}
 					/>
-					<TextField
-						size='small'
-						label='Simulation software'
-						value={simulator}
-						disabled
-					/>
+					<FormControl
+						fullWidth
+						sx={{
+							maxWidth: ({ spacing }) => `calc(100vw - ${spacing(14)})`,
+							width: '380px'
+						}}>
+						<InputLabel id='simulator-select-label'>Simulation software</InputLabel>
+						<Select
+							labelId='simulator-select-label'
+							size='small'
+							label='Simulation software'
+							defaultValue={forwardedSimulator}
+							onChange={evn => setSelectedSimulator(evn.target.value as SimulatorType)}
+						>
+							<MenuItem value={SimulatorType.SHIELDHIT}>SHIELD-HIT12A</MenuItem>
+							<MenuItem value={SimulatorType.FLUKA}>Fluka</MenuItem>
+						</Select>
+					</FormControl>
```

Comment:
It would be useful to lock this form control when we have forwardedSimulator defined and simulationSourceType is set to 'files'